### PR TITLE
fix(openapi): use provided from openapi message parameters models

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Follow-up to #11802
  * migrate messagesService to TS
  * migrate reactionsService to TS
  * move message methods to messagesService, move translation methods to translation service

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "@nextcloud/eslint-config": "^8.3.0",
     "@nextcloud/stylelint-config": "^2.4.0",
     "@nextcloud/webpack-vue-config": "^6.0.1",
+    "@types/crypto-js": "^4.2.2",
```
Comment (DorraJaouad) [inline package.json:85]: Why are you adding it? 
According to their repo, there is a native Crypto in browsers now 
https://github.com/brix/crypto-js#discontinued

File: src/types/index.ts [hunk 2]
```diff
@@ -45,21 +45,28 @@ export type Mention = ParamObject & {
...
-	'size': number,
-	'path': string,
-	'link': string,
+export type File = Pick<components['schemas']['RichObjectParameter'], 'size'|'path'|'link'|'mimetype'|'preview-available'> & {
 	'etag': string,
 	'permissions': number,
-	'mimetype': string,
-	'preview-available': string,
 	'width': number,
 	'height': number,
```
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/types/index.ts:52]: And type, right 🙈. It was in ParamObject

File: package.json (hunk 1 above)
Comment (Antreesy) [reply package.json:85]: Then it's time to migrate to native methods)

**Review by DorraJaouad — APPROVED** [review]
All good

**Pull request merged**